docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md

```diff
@@ -205,6 +205,12 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
 | Transaction                 | Expired Transactions: the number of expired distributed transactions.<br>Transaction Conflicts: the number of conflicts detected among uncommitted distributed transactions.<br><br>This is related to the process that resolves conflicts for write transactions. This process reads all intents that could conflict and tries to abort transactions with a lower priority. If a write transaction conflicts with a higher-priority transaction, then an error is returned and this metric is iterated. | Alerting is not required.                                    | ![Transaction](/images/yp/metrics75.png)  |
 | Transaction pool cache hits | Percentage of transaction pool requests fulfilled by the transaction pool cache. | This metric is informational and should not be subject to alerting. You may consider this information while examining alerts on other metrics. | ![Transaction pool cache hits](/images/yp/metrics105.png) |
 
+## View fine grained metrics
+
+YugabyteDB Anywhere allows you to view granular data points in metric charts by selecting a portion of time range along the y-axis of the graph. To do so, navigate to **Universes > Universe-Name > Metrics** and select a portion of any graph to view the granular results as per the following illustration:
+
+![Fine grained metrics](/images/yb-platform/fine-grain-metrics.png)
```

Comment:
when taking screenshots, try to size the window to the minimum size that shows what we want to show. There is a bunch of whitespace to the right of the 2 charts